apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notice.js

```diff
@@ -0,0 +1,15 @@
+import { Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const GlobalStylesNotice = () => {
+	return (
+		<Notice status="warning" isDismissible={ false }>
+			{ __(
+				"Your style changes won't be public until you upgrade your plan. Uou can revert your styles.",
```

Comment:
Typo change `Uou` to `You`.

Reply:
This code is also missing the `upgrade your plan` and `revert your styles` links. It's fine to add them in a follow-up.